packages/cli/src/commands/teams/command.ts

```diff
@@ -0,0 +1,77 @@
+import { Command } from '../help';
+import { packageName } from '../../util/pkg-name';
+
+export const teamsCommand: Command = {
+  name: 'teams',
+  description: 'Manage teams under your Vercel account',
+  arguments: [],
+  subcommands: [
+    {
+      name: 'add',
+      description: 'Create a new team',
+      arguments: [],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'ls',
+      description: "Show all teams you're a part of",
+      arguments: [],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'switch',
+      description: 'Switch to a different team',
+      arguments: [
+        {
+          name: 'name',
+          required: false,
+        },
+      ],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'invite',
+      description: 'Invite a new member to a team',
+      arguments: [
+        {
+          name: 'emails',
+          required: true,
+        },
+      ],
+      options: [],
+      examples: [],
+    },
+  ],
+  options: [
+    {
+      name: 'next',
+      shorthand: 'n',
```

Comment:
Hm, other `--next` commands use `-n` though. Maybe we keep the `-N`, but i add another shorthand line for `-n` so that we are consistent across commands?

Reply:
It looks like `-N` everywhere in practice, but it was documented as `-n` in some: https://github.com/vercel/vercel/pull/10432#pullrequestreview-1610392313

In use, I can only find `-N`:

```
8 results - 8 files

packages/cli/src/commands/secrets.js:
  120      '-y': '--yes',
  121:     '-N': '--next',
  122    });

packages/cli/src/commands/alias/index.ts:
  27        '-y': '--yes',
  28:       '-N': '--next',
  29      });

packages/cli/src/commands/certs/index.ts:
  34        '--next': Number,
  35:       '-N': '--next',
  36        '--limit': Number,

packages/cli/src/commands/dns/index.ts:
  25        '--next': Number,
  26:       '-N': '--next',
  27        '--limit': Number,

packages/cli/src/commands/domains/index.ts:
  96        '--next': Number,
  97:       '-N': '--next',
  98        '-y': '--yes',

packages/cli/src/commands/list/index.ts:
  33        '--next': Number,
  34:       '-N': '--next',
  35        '--prod': Boolean, // this can be deprecated someday

packages/cli/src/commands/project/index.ts:
  24        '--next': Number,
  25:       '-N': '--next',
  26      });

packages/cli/src/commands/teams/list.ts:
  19      '-C': '--count',
  20:     '-N': '--next',
  21    });
```

I think this should be documented as `-N` everywhere to match existing behavior.